mindsdb_native/libs/phases/data_cleaner/data_cleaner.py

```diff
@@ -3,28 +3,32 @@
 
 
 class DataCleaner(BaseModule):
-    def _cleanup_w_missing_targets(self, df):
+    def get_empty_columns(self, df):
```

Comment:
"private" methods should use `_` in front, at least that's the style we went with